src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -24,22 +24,91 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+declare(strict_types=1);
+
 namespace PrestaShopBundle\Controller\Admin\Sell\Catalog;
 
 use Exception;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureConstraintException;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureNotFoundException;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Query\GetFeatureForEditing;
+use PrestaShopBundle\Bridge\AdminController\Action\HeaderToolbarAction;
+use PrestaShopBundle\Bridge\AdminController\Action\ListBulkAction;
+use PrestaShopBundle\Bridge\AdminController\Action\ListHeaderToolbarAction;
+use PrestaShopBundle\Bridge\AdminController\Action\ListRowAction;
+use PrestaShopBundle\Bridge\AdminController\AdminControllerTrait;
+use PrestaShopBundle\Bridge\AdminController\ControllerBridgeInterface;
+use PrestaShopBundle\Bridge\AdminController\ControllerConfiguration;
+use PrestaShopBundle\Bridge\AdminController\Field\Field;
+use PrestaShopBundle\Bridge\Helper\HelperListConfiguration;
+use PrestaShopBundle\Bridge\Helper\HelperListCustomizer\HelperListFeatureBridgeCustomizer;
+use PrestaShopBundle\Bridge\Smarty\SmartyTrait;
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
 use PrestaShopBundle\Security\Annotation\AdminSecurity;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Tools;
 
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController
+class FeatureController extends FrameworkBundleAdminController implements ControllerBridgeInterface
```

Comment:
As discussed, It may be worth it to create 3 interfaces, 1 for list related methods (like `getListId()`), one for more global methods (maybe like `getTheme()`?) and the actual `ControllerBridgeInterface`